apps/challenges/serializers.py

```diff
@@ -102,11 +104,23 @@ class ZipChallengeSerializer(ChallengeSerializer):
     """
     Serializer used for creating challenge through zip file.
     """
+    def __init__(self, *args, **kwargs):
+        super(ZipChallengeSerializer, self).__init__(*args, **kwargs)
+
+        context = kwargs.get('context')
+        if context:
+            image = context.get('image')
+            evaluation_script = context.get('evaluation_script')
+
+        kwargs['data']['image'] = image
+        kwargs['data']['evaluation_script'] = evaluation_script
+
+
     class Meta:
         model = Challenge
         fields = ('id', 'title', 'short_description', 'description', 'terms_and_conditions',
-                  'submission_guidelines', 'start_date', 'end_date', 'creator',
-                  'published', 'enable_forum', 'anonymous_leaderboard', 'is_active',)
+                  'submission_guidelines', 'start_date', 'end_date', 'creator', 'evaluation_details',
```

Comment:
@RishabhJain2018 can you please tell me what is the significance of `evaluation_details` here? I might be missing something here.